e2e/app/page/workspace-analysis-page.ts

```diff
@@ -41,7 +42,11 @@ export default class WorkspaceAnalysisPage extends WorkspaceBase {
     const appSelectorButton = Button.findByName(this.page, { normalizeSpace: 'Choose an App' });
     await appSelectorButton.click();
 
-    const appSelectorOptions = SelectMenu.findByName(this.page, { normalizeSpace: 'Choose One' });
+    const appSelectorOptions = SelectMenu.findByName(this.page, {
+      type: ElementType.Dropdown,
+      name: 'Select an application',
+      ancestorLevel: 2
+    });
```

Comment:
Switched to how we select options on the [create-account-page](https://github.com/all-of-us/workbench/blob/main/e2e/app/page/create-account-page.ts#L43-L47), passes locally for me